spec/ParseQuery.spec.js

```diff
@@ -3266,8 +3266,8 @@ describe('Parse.Query testing', () => {
       );
   });
 
-  it('select keys query', function(done) {
-    const obj = new TestObject({ foo: 'baz', bar: 1 });
+  it('select/unselect keys query', function(done) {
```

Comment:
Can you separate into its own test? Also a test for checking that exclude has precedence over select and include and include all?an